File: homeCSharp/view/Menu.cs
namespace GeekBrains;

public class Menu {

    private static UserInputFromConsole scan = new UserInputFromConsole();
    private static String userSelect = "EMPTY";
    private static ToPresenter toPresenter = new ToPresenter();

    public void Start(){

        while (userSelect != "q"){
            userSelect = "EMPTY";
            userSelect = scan.inputKey(TextsMenu.inviteMenu);
            toPresenter.doSomething(userSelect);           
        }
        Console.WriteLine("Пока!");
    }
}

File: hw6/src/view/Menu.java
package view;

import presenter.ToPresenter;

public class Menu {
    private View scn = new View();
    private String userSelect = "EMPTY";
    private ToPresenter pres = new ToPresenter();
    public void start(){
        while (!userSelect.equals("q")){
            userSelect = "EMPTY";
            userSelect = scn.inputKey(TextMenu.menu);
            pres.doSomething(userSelect);
        }
        scn.scn.close();
        System.out.println("Bye");
    }
}
